src/DynamoCore/Properties/Resources.en-US.resx

```diff
@@ -810,4 +810,25 @@ This package has not been loaded because another conflicting package was loaded
   <data name="NodeUnhandledMsg" xml:space="preserve">
     <value>Unhandled 'DummyNode.NodeNature' value: {0}</value>
   </data>
+  <data name="CoreLibraryLoadFailureForBlockedAssembly" xml:space="preserve">
+    <value>{0}
+
+This assembly is likely blocked. Try restarting Dynamo after unblocking this assembly and all other core assemblies that might be blocked. If you are running a downloaded DynamoSandbox build that is extracted from a .zip file, try using 7zip to extract the Dynamo binaries and try again.</value>
```

Comment:
I wonder if a regular user will know how to unblock an assembly ..
I guess they can google it
Also maybe the zip thing is the most common reason..

Reply:
we try to document it here:
https://github.com/DynamoDS/Dynamo/wiki/How-to-Utilize-Dynamo-Builds